templates/user/dashboard/repolist.tmpl

```diff
@@ -109,10 +109,12 @@
 						{{.i18n.Tr "forks"}}
 						<div v-show="reposFilter === 'forks'" class="ui circular mini grey label">${repoTypeCount}</div>
 					</a>
+					{{if not .DisableNewPullMirrors}}
```

Comment:
It just disabled new pull mirror but not display old mirrors.

Reply:
According to https://github.com/go-gitea/gitea/pull/13084 disabling mirror feature should still allow admin to migrate existing mirrors to regular repos so why "old" mirrors should be still displayed.

Reply:
But it just disable creating new pull mirrors but not disable mirrors.

Reply:
Will changing `not .DisableNewPullMirrors` to `.MirrorsEnabled` here be ok for you?

Reply:
Done in b1c36319891a8729e54f04681107e1f1ec5d38a5